_posts/2022-07-15-blog-wpe-graphics-architecture.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "WPE Graphics architecture"
+tags: [blogpost]
+author: magomez
+permalink: /blog/03-wpe-graphics-architecture.html
+---
+
+Following <a href="/blog/02-overview-of-wpe.html">the previous post in the series about <a href="https://wpewebkit.org/">WPE</a> where we talked about the <a href="https://wpewebkit.org/">WPE</a> components, this post will explain briefly the <a href="https://wpewebkit.org/">WPE</a> graphics architecture, and how the engine is able to render HTML content into the display. If you haven't read the previous entries in this blog post series about <a href="https://wpewebkit.org/">WPE</a>, we recommend you to start with the <a href="/blog/01-happy-birthday-wpe.html">first post in the series</a> for an introduction, and then come back to this.
```

Comment:
We should probably try to use the `url` template operator everywhere to make internal links and image embedding work in all cases (e.g. like previews). Anyway, I would rather publish the article now as-is, and do a pass later on to amend things.